core/os/os2/process_linux.odin

```diff
@@ -71,25 +302,446 @@ _Sys_Process_Attributes :: struct {}
 
 @(private="package")
 _process_start :: proc(desc: Process_Desc) -> (process: Process, err: Error) {
+	has_executable_permissions :: proc(fd: linux.Fd) -> bool {
+		backing: [48]u8
+		_ = fmt.bprintf(backing[:], "/proc/self/fd/%d", fd)
+		return linux.access(cstring(&backing[0]), linux.X_OK) == .NONE
+	}
+
+	TEMP_ALLOCATOR_GUARD()
+
+	if len(desc.command) == 0 {
+		return process, .Invalid_File
```

Comment:
`.Invalid_File` doesn't really fit this condition, possibly need a new error here like `.Invalid_Command`